tools/codegen/pkg/modelgen/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
as soon as this stuff is in, I can start work on the BUILD for our "custom language" using the proto libs and we can have the tests run via bazel.